Yoast/javascript
# Add Norwegian stemmer

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Norwegian stemmer.
* [Yoast SEO Premium] Introduces word form support for Norwegian.

## Relevant technical choices:

* There are still suffixes that are not yet covered by this stemmer, simple past and past participle suffixes -de, -te, -d, -t and adjective suffixes -ere, -are, -est, -este, -ast, -aste, -t. 
* Words that end in specific suffixes that overlap with each other, e.g. `kjende` -> `kjend` + `e` are not stemmed correctly. However, this word would be recognized as having suffix `-ende` instead of only `-e`. As a result, the word would not be stemmed at all because `-ende` is not in R1.
* Since there is no exception list implemented, words that end in an ending that looks like a valid suffix will be incorrectly stemmed. E.g. arbeidstaker, finansier, konkurrans etc.

## Test instructions

This PR can be tested by following these steps:

* Add more words to spec file and delete them after testing.
* In content analysis app:
    * Turn the morphology feature on and add minimum 300 words to the text field
    * Set the locale to "nb"
    * Use `skikkelse` as the focus keyphrase 
    * Add these other forms of the word in the text field `skikka, skikke, skikkelig, skikkelse, skikkelsen, skikkelser, skikker, and skikket` and make sure that all these forms are recognized.
    * Repeat the same process with different form in the focus keyphrase and make sure that the other forms are still recognized as well. 

## QA test instruction
* In your testing environment, set the language to Norwegian
* Make sure to test this functionality in premium
* Add text minimum 300 words
* Use `skikke` as the focus keyphrase 
* Add these other forms of the word in the text field `skikka, skikke, skikkelig, skikkelse, skikkelsen, skikkelser, skikker, and skikket` and make sure that all these forms are recognized.
* Repeat the same process with different form in the focus keyphrase and make sure that the other forms are still recognized as well. 

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #https://yoast.atlassian.net/browse/LINGO-350
